docs/external_api.md

```diff
@@ -0,0 +1,1526 @@
+## Params
+
+```per_page=10  # default 10```
+
+```page=5       # default 1```
+
+```updated_after=Thu, 26 May 2022 14:35:09 +0100       # default nil```
+
+## Endpoint
+
+```https://www.naturecommitments.org/api/v1/commitments?per_page=10&page=5&updated_after=Thu, 26 May 2022 14:35:09 +0100```
+
+
+## JSON structure
+
+```json
+[
+  {
+    "id": 11,
+    "latitude": 55.37911,
+    "longitude": -110.308466,
+    "current_area_ha": 0,
+    "committed_year": "2020",
+    "implementation_year": "2021",
+    "name": "The name of the commitment",
+    "description": "A description of the commitment",
+    "stage": "A string, either In progress/Committed/Implemented fully",
+    "responsible_group": "A string describing the responsible group",
+    "duration_years": "more than 25 years",
+    "area_owner_and_role": "A string describing the owner and their role if the are does not belong to the group/person making the commitment",
+    "created_at": Wed, 18 May 2022 14:43:33 UTC +00:00,
+    "updated_at": Wed, 18 May 2022 14:43:33 UTC +00:00,
+    "managers": [
+      {
+        "id": 2,
+        "name": "Indigenous peoples"
+      }
+    ],
+    "actions": [
+      {
+        "id": 4,
+        "name": "Education or awareness-raising"
+      }
+    ],
+    "countries": [
+      {
+        "id": 38,
+        "name": "Canada",
+        "iso": "CA"
+      },
+      {
+        "id": 233,
+        "name": "United States of America",
+        "iso": "US"
+      }
+    ],
+    "objectives": [
+      {
+        "id": 1,
+        "name": "Sustainable use"
+      }
+    ],
+    "threats": [
+      {
+        "id": 14,
+        "name": "Invasive species"
+      }
+    ],
+    "links": [
+      {
+        "id": 40,
+        "url": "https://www.a-url.com"
+      }
+    ]
+  }
+]
+```
+
+## Associated records
```

Comment:
What's the reason for providing this information if there aren't index endpoints specifically for these (they only appear as associations in the commitments endpoint?)?

Reply:
It's so they have a full list of the categories they have to handle in the import. as we did for them, CBD have to map these to their own categories.

Reply:
Makes sense - would it be useful to add a line to the effect of "The data below are provided for reference and are included in the API where associated with a particular commitment - the API does not provide an index of these associations" just in case someone other than CBD comes across this and thinks they might be endpoints (even though that's already mentioned in the main README)?

Reply:
added